src/main/webapp/view/discussion/discussion-comment.js

```diff
@@ -134,6 +136,18 @@ export class DiscussionComment extends HTMLElement {
     const offset = elementTop - scrollPaneTop;
     ELEMENT_DISCUSSION.scrollTop = offset;
   }
+
+  /**
+   * Sets if this comment should be highlighted on the DOM.
+   */
+  setHighlighted(highlight) {
```

Comment:
I think per the single responsibility principle, it might be better to have separate functions for removing and adding the highlights.

Reply:
Ah yes, split it!